java/openjdk10/Portfile

```diff
@@ -38,8 +57,10 @@ build {}
 destroot.violate_mtree yes
 
 destroot {
-    ui_msg "\nWarning: Support for OpenJDK 10.x has reached end of life and there will be no more updates."
-    ui_msg "         Please consider upgrading to a supported OpenJDK version.\n"
+    if {${subport} eq ${name}} {
+        ui_msg "\nWarning: Support for OpenJDK 10.x has reached end of life and there will be no more updates."
+        ui_msg "         Please consider upgrading to a supported OpenJDK version.\n"
+    }
```

Comment:
That message should be in `notes {}` instead.

Reply:
I fixed that here: https://github.com/macports/macports-ports/pull/2672

Reply:
It appears to be beyond my git foo to figure out how to merge the changes into my branch since this was already committed to master.